backend/src/akvo/lumen/http.clj

```diff
@@ -0,0 +1,38 @@
+(ns akvo.lumen.http
+  "wrapping http client calls so we have robust settings values"
+  (:require
+   [clj-http.client :as client]
+   [clj-http.conn-mgr :as http.conn-mgr]))
+
+(defn- default-connection-manager-opts
+  ":timeout - Time that connections are left open before automatically closing ... in seconds" 
+  [opts]
+  (merge {:timeout 10 :threads 2 :insecure? false :default-per-route 2}
+         (select-keys opts [:timeout :threads :insecure? :default-per-route])))
+
+(defn new-connection-manager
+  ([] (new-connection-manager {}))
+  ([opts]
+   (http.conn-mgr/make-reusable-conn-manager (default-connection-manager-opts opts))))
+
+
+(defn shutdown-manager [connection-manager]
+    (http.conn-mgr/shutdown-manager connection-manager))
+
+(def default-req-opts
+  ":connection-timeout - time to establish the socket
+   :connection-request-timeout - time to have a free socket
+   :socket-timeout - time to get the response" 
+  {:connection-timeout 2000 :connection-request-timeout 50 :socket-timeout 5000})
+
+(defn get* [url & [opts]]
```

Comment:
why `get*` and not `get`? And the same for others

Reply:
i don't want to exclude core/get so i have to remember to use core/get each time i need it inside the namespace